crates/apub/src/api/user_settings_backup.rs

```diff
@@ -133,7 +137,8 @@ pub async fn import_settings(
     local_user_view.local_user.id,
     &local_user_form,
   )
-  .await?;
+  .await
+  .ok();
```

Comment:
You'll probably see a conflict anyway after the other one gets merged, but the `.ok()` should be removed.

Reply:
bump on this one, then we can merge.

Reply:
Fixed and merged.